PowerEditor/src/WinControls/TreeView/TreeView.h

```diff
@@ -40,7 +40,7 @@ class TreeView : public Window {
 
 	virtual void init(HINSTANCE hInst, HWND parent, int treeViewID);
 	virtual void destroy();
-	HTREEITEM addItem(const TCHAR *itemName, HTREEITEM hParentItem, int iImage, LPARAM lParam = NULL);
+	HTREEITEM addItem(const TCHAR *itemName, HTREEITEM hParentItem, int iImage, LPARAM lParam = 0);

```

Comment:
For WPARAM and LPARAM prefer "zero" instead of `NULL`. MS documentation is using mostly zero for them.